### PR TITLE
CB-74: Remove all warnings from test

### PR DESCRIPTION
**Pull request by jomadoye**
# JIRA TICKET NAME:
CB-74: [Fix warning on tests](https://issues.openmrs.org/browse/CB-74)

## SUMMARY:
This PR fixes all the warnings that show when running tests

### CODE REVIEW

**Comment by dkayiwa** [comment]
@andela-jomadoye hopefully you did take a look at this: https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by jomadoye** [comment]
@dkayiwa Yes I did.

**Comment by dkayiwa** [comment]
@andela-jomadoye did you follow the instructions there in regards to what to do when a ticket is ready for review?

File: app/js/components/tabs/tabcomponents/drugOrderComponent.jsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ class DrugOrderComponent extends Component {
             });
         this.props.fetchData('concept?name=REASON%20ORDER%20STOPPED')
             .then(conceptReason => {
-                this.props.fetchData(`concept/${conceptReason.results[0].uuid}`)
+                conceptReason.results && this.props.fetchData(`concept/${conceptReason.results[0].uuid}`)
```
[the remark this answers is not in the document]
Comment (dkayiwa) [reply app/js/components/tabs/tabcomponents/drugOrderComponent.jsx:41]: @andela-jomadoye i just wanna make sure that you did see these automated reviews. Did you?

**Pull request closed without merging**